README.md

```diff
@@ -43,7 +43,7 @@ documentation](https://stevedonovan.github.io/winapi/api.html); this is generate
 
 This is straightforward; the only external dependency is
 [Penlight](https://github.com/lunarmodules/Penlight), which in turn needs
-[LuaFileSystem](https://keplerproject.github.io/luafilesystem/). These are already present
+[LuaFileSystem](https://lunarmodules.github.io/luafilesystem/) These are already present
```

Comment:
```suggestion
[LuaFileSystem](https://lunarmodules.github.io/luafilesystem/). These are already present
```

Thanks for the link fix, but why break the sentence punctuation in the process?

Reply:
oh, sorry.